datastore/src/main/java/com/ds/datastore/Book.java

```diff
@@ -15,8 +13,17 @@
     private String description;
     private Language language;
 
+    private Long store_id;
+
     public Book() {}
 
+    public Long getStore_id() {
```

Comment:
The convention in Java is camel case so it should be `getStoreId`

Reply:
But moreover, I think we will it will be a lot easier if we use bi-diretional OneToMany as described in that section here:

https://vladmihalcea.com/the-best-way-to-map-a-onetomany-association-with-jpa-and-hibernate/

where we have a reference to a `Bookstore` and not an `id`